qulice-pmd/src/test/resources/com/qulice/pmd/StaticPublicVoidMainMethod.java

```diff
@@ -0,0 +1,12 @@
+package foo;
+
+public class StaticPublicVoidMainMethod {
+
+    public static void main(final String... args) {
```

Comment:
@HDouss We have another rule com.mebigfatguy.fbcontrib.detect.UseVarArgs, that restricts us declare `public static void main(String args[])` or  `public static void main(String[] args)`. There is no need to check array as parameter.